tutorials/3d/environment_and_post_processing.rst

```diff
@@ -391,11 +598,149 @@ and Saturation:
 
-.. image:: img/environment_adjusments_default_gradient.png
+A linear black-to-white gradient like the following one will produce no effect:
+
+.. image:: img/environment_adjustments_default_gradient.png
 
 But creating custom ones will allow to map each channel to a different color:
 
-.. image:: img/environment_adjusments_custom_gradient.png
+.. image:: img/environment_adjustments_custom_gradient.png
+
+A 3D look-up-texture (LUT) can also be used for color correction. This is a
+special texture used to modify each color channel separately from one another
+(red, green, blue). This image can be of any resolution, but since color
+correction is low-frequency data, sticking to low resolutions is recommended for
+performance reasons. A LUT texture's resolution is typically 17×17×17, 33×33×33,
+51×51×51 or 65×65×65 (the odd size allows for better interpolation).
+
+.. note::
+
+    Color data within the LUT must be stored in linear space (not
+    perceptual/sRGB), as Godot performs 3D rendering in linear space.
+
+For this to work, the look-up texture's import mode must be set to Texture3D
+in the Import dock (instead of being imported as a regular Texture2D):
+
+.. image:: img/environment_adjustments_3d_lut_import.png
+
+Make sure to configure the number of horizontal and vertical slices to import as
+well. If this is not done, the LUT texture will not affect the viewport
+correctly when used. You can preview how the 3D texture was imported by
+double-clicking it, in the FileSystem dock, then going to the inspector to flip
+through the texture's layers.
+
+You can use this neutral 33×33×33 LUT template as a base (right-click and choose
+**Save as…***):
+
+.. image:: img/environment_adjustments_3d_lut_template.png
+
+With the above LUT template, after changing its import mode to **Texture3D**,
+set its number of **Horizontal** slices to ``33`` in the Import dock then click
+**Reimport**. If you load this LUT into the **Color Correction** property, you
+won't see any visible difference for now since this texture is designed to be a
+neutral starting point.
+
+This LUT template can be modified in an image editor to provide a different
+mood to the image. A common workflow is to place the LUT image next to a
+screenshot of the project's 3D viewport, then use an image editor to modify both
+the LUT image and the screenshot at the same time. The LUT can then be saved and
+applied to the game engine to perform the same color correction in real-time.
+
+For example, with the LUT template modified in an image editor to give it a
+"sepia" look, the result on the right is obtained:
+
+.. image:: img/environment_adjustments_3d_lut_comparison.png
+
+.. note::
+
+    Adjustments and color correction are applied *after* tonemapping.
+    This means the tonemapping properties defined above still have an effect
+    when adjustments are enabled.
+
+Camera attribute options
+------------------------
+
+Depth of Field / Far Blur
+^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
There should be a mention here that when using CameraAttributesPhysical, depth of field is calculated automatically from focus distance, focal length, and aperture